geopyspark/geotrellis/histogram.py

```diff
@@ -0,0 +1,162 @@
+"""This module contains the ``Histogram`` class which is a wrapper of the GeoTrellis Histogram
+class.
+"""
+
+
+class Histogram(object):
+    """A wrapper class for a GeoTrellis Histogram.
+
+    The underlying histogram is produced from the values within a
+    :class:`~geopyspark.geotrellis.layer.TiledRasterLayer`. These values represented by the
+    histogram can either be ``Int`` or ``Float`` depending on the data type of the cells in the
+    layer.
+
+    Args:
+        scala_histogram (py4j.JavaObject): An instance of the GeoTrellis histogram.
+
+    Attributes:
+        scala_histogram (py4j.JavaObject): An instance of the GeoTrellis histogram.
+    """
+
+    def __init__(self, scala_histogram):
+        self.scala_histogram = scala_histogram
+
+    @classmethod
+    def from_tiled_layer(cls, tiled_layer):
```

Comment:
Although you can't do it right now you should be able to get a histogram from `RasterLayer`. I would argue that we don't want to proliferate `from_*` for geopyspark classes that have `.get_histogram` methods.